modules/Swing.lua

```diff
@@ -160,13 +160,30 @@ function Swing:STOP_AUTOREPEAT_SPELL()
 	end
 end
 
+function IsClassSwingAbility(spellName)
+	if playerclass == "WARRIOR" then
+		if spellName == "Heroic Strike" then
```

Comment:
Ideally you would use GetSpellInfo above the function to lookup the localized names of the spells in question, so that it works on non-english clients.

ie. something like:
local HEROIC_STRIKE = GetSpellInfo(78)
local MAUL = GetSpellInfo(6807)